$NetBSD: patch-docs__makefile,v 1.1 2013/07/07 23:46:50 marino Exp $

Disable PDF documentation generation; it pulls way too much in.

--- docs/makefile.orig	2013-07-03 01:11:54.000000000 +0000
+++ docs/makefile
@@ -116,10 +116,9 @@ build_doc: $(APIFILES) aws_docs sg_docs
 	echo AWS Documentation built with success.
 	${MAKE} -C ../templates_parser doc
 
-aws_docs: aws.texi aws.pdf aws.html aws.txt aws.info
+aws_docs: aws.texi aws.html aws.txt aws.info
 
-sg_docs: style-guide.pdf style-guide.html style-guide.txt \
-	style-guide.info
+sg_docs: style-guide.html style-guide.txt style-guide.info
 
 setup:
 
